src/routes/safe/components/Balances/SendModal/screens/ContractInteraction/utils/index.ts

```diff
@@ -63,6 +63,13 @@ export const isInt = (type: string): boolean => type.indexOf('int') === 0
 export const isByte = (type: string): boolean => type.indexOf('byte') === 0
 
 export const isArrayParameter = (parameter: string): boolean => /(\[\d*])+$/.test(parameter)
+export const getArrayParameterIfArray = (parameter: string): unknown[] | null => {
+  try {
+    return JSON.parse(parameter)
+  } catch (err) {
+    return null
```

Comment:
Should we show the error ? If not,we could remove the catch statement since it seems useless.

Reply:
I think it's not needed for the goal of the function. But it looks a bit misleading as if I pass a valid JSON it'll return the parsed JSON.

I'm not sure if we need to rename the method or evaluate that the parsed value is an array.

@Agupane?

Reply:
I renamed it to `getParsedJSONOrArrayFromString` what do you think? feel free to propose another name, naming is difficult